pkg/objstore/azure/helpers.go

```diff
@@ -104,7 +106,7 @@ func getContainerURL(ctx context.Context, logger log.Logger, conf Config) (blob.
 		retryOptions.TryTimeout = time.Until(deadline)
 	}
 
-	dt, err := DefaultTransport(conf)
+	dt, err := exthttp.DefaultTransport(exthttp.HTTPConfig)
```

Comment:
I think this would be `conf`! `exthttp.HTTPConfig` is a type, so you might need to change the argument type in `getContainerURL`.

Reply:
Yes. Just did that